### DISCUSSION TITLE
Customize the wine version

### DISCUSSION THREAD

**Discussion by msmurphy** [netbrain/zwift#70] [Ideas]
I used to run zwift via lutris and wine in the past but it became too much of a headache to maintain.   Currently can't get it working because the webview2 installer won't work.     I never had performance issues with it though.   Running your docker container and zwift is pegging my cpu at 100% load and it just doesn't run fast enough.   I'm on a I7-6700k which is pretty old but should be able to handle this.    I've never had any issues running through lutris, but I did use a custom wine verison (https://github.com/GloriousEggroll/wine-ge-custom).  

I'd like to use that again with your container in the hopes that it fixes my performance issues.

**Comment by netbrain** [comment]
Couldn't agree more! Would love to incorporate some customization with this in the future! :crossed_fingers: 

Would probably relate to: https://github.com/netbrain/zwift/issues/28
